bedrock/firefox/views.py

```diff
@@ -967,3 +972,14 @@ def firefox_welcome_page1(request):
     template_name = 'firefox/welcome/page1.html'
 
     return l10n_utils.render(request, template_name, context)
+
+
+def firefox_mobile(request):
+    locale = l10n_utils.get_locale(request)
+
+    if lang_file_is_active('firefox/mobile-2019', locale):
+        template_name = 'firefox/mobile-en.html'
```

Comment:
Not a blocker, but if we're doing L10n now it would be nice to rename this template to something other than `mobile-en`.

Reply:
switching the name to mobile-2019